### PR TITLE
Various typos

### PR DESCRIPTION
**Pull request by piotrdrag**
Fixes also issue #5.

### CODE REVIEW

File: apps/settings/settings.properties [hunk 1]
```diff
@@ -341,7 +341,7 @@ delete-apn-confirm=Usunąć ten APN?
 delete-apn=Usuń
 add-apn=Dodaj nowy APN
 reset-apn=Przywróć domyślny
-apnSettings-reset=Reset APN-u
+apnSettings-reset=Reset APN
```
Comment (splewako) [inline apps/settings/settings.properties]: Why?

Comment (piotrdrag) [inline apps/settings/settings.properties]: Sounds better to me. I can revert if you want.

Comment (splewako) [inline apps/settings/settings.properties]: It is status message, maybe "Resetowanie APN-u…" (or even without APN)?

**Comment by piotrdrag** [comment]
I've reverted the "APN-u" changes.

**Comment by splewako** [comment]
Thank you for the patch

**Pull request closed without merging**